app/build.gradle

```diff
@@ -48,9 +50,25 @@ dependencies {
     annotationProcessor libraries.immutablesValue // <-- for annotation processor
     compileOnly libraries.immutablesValue // <-- for annotation API
     compileOnly libraries.immutablesGson // for annotations
-    implementation 'com.nytimes.android:store3:3.0.1'
-    implementation 'com.nytimes.android:cache3:3.0.1'
-    implementation 'com.nytimes.android:middleware3:3.0.1'
-    implementation 'com.nytimes.android:filesystem3:3.0.1'
+    implementation project(':store')
```

Comment:
might need to flip before checking in....

Reply:
will do once I can push the new snapshot thanks!